### ISSUE TITLE
Icon issue when installing Komodo on a shared folder

### ISSUE THREAD

**Issue by llgxela** [Komodo/KomodoEdit#1928]
### Short Summary

When installing Komodo Edit on a Windows shared folder, many icons do not show up properly and are instead replaced by the 'sad face' icon placeholder.

### Steps to Reproduce

a] Install Komodo Edit on a Windows shared folder, let's say C:\apps$. Permission do not seem to matter;
b] Create a link to the main Komodo Edit executable and place it in another folder, for instance C:\links;
c] Edit the link properties and change the path to UNC. For example, change C:\apps$\\...\komodo.exe to \\192.168.0.44\apps$\\...\komodo.exe;
d] Launch the program by double-clicking on the newly-edited link.

### Expected results

All icons should be rendered properly on screen.

### Actual results

Many icons, particularly the ones on the left pane show a placeholder icon instead of the correct one.

### Platform Information

*Komodo Edit or IDE?*  Komodo Edit
*Komodo Version?* Any version
*Operating System (and version)?* Windows 7 Professional x86_64

### Additional Information

None unfortunately, I can provide screenshots if the issue is not clear though.

**Comment by Naatan** [comment]
What Komodo version?

**Comment by jimtut** [comment]
Komodo IDE, version 10.0.1, build 89237, platform win32-x86.
Built on Wed Jun 15 10:57:56 2016.

**Comment by llgxela** [comment]
Komodo Edit 9.2.1 and 10.0.1.

Thank you!

**Comment by Naatan** [comment]
And everything else works properly? I'm hesitant to provide support for launching Komodo over a UNC path, but if all that needs fixing is the icons then it's worth looking into.

**Comment by llgxela** [comment]
Yes, as far as my testing goes everything else works fine. Please, let me know if you need further assistance.

**Comment by Naatan** [comment]
thanks, I'll look into this for 11. Unfortunately I cannot give you a quick fix for this as it's very niche and not something that we explicitly support. Perhaps you could try mounting it as a network drive in the meantime.
